spec-examples/src/chacha20_poly1305/chacha20.rs

```diff
@@ -8,17 +8,17 @@ bytes!(Key, 32);
 
 pub fn state_to_bytes(x: State) -> StateBytes {
     let mut r = StateBytes::new();
-    for i in 0..x.len() {
+    for i in 0usize..x.len() {
```

Comment:
I'm don't really like this, but we agreed that something like this is necessary. So lets go with ti.